.drone.yml

```diff
@@ -0,0 +1,40 @@
+pipeline:
+
+  web_build_image:
+    image: docker:17.07.0-ce
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    commands:
+      - cd platform-hub-web
```

Comment:
cd to platform-hub-web? Drop this

Reply:
haha you can see where I get my inspiration from XD